jjb/mkck/mkck-trigger.yaml

```diff
@@ -180,16 +222,9 @@
                 - snippets/mkck-target-properties.bash
         - inject:
             properties-file: 'target.properties'
-        - shell: "git clone https://github.com/suse/sesci"
-        - shell: |
-            virtualenv v
-            . v/bin/activate
-            pip install -U pip
-            pip install python-openstackclient openstacksdk docopt PyYaml paramiko
-        - shell: |
-            . v/bin/activate
-            STATUS=$PWD/mkck.status
-            cd sesci && OS_CLOUD=$TARGET_CLOUD python -u os-server.py -a create -f conf/$CONF.yaml -s $STATUS
+        - git-clone-sesci-suse-master
+        - create-openstackclient-venv
```

Comment:
That's not the openstackclient-venv, I would say it is 'sesci' env

Reply:
done